files/signals.py

```
@@ -0,0 +1,34 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+from google.api_core import exceptions as gcs_exceptions
+from loguru import logger
+
+from firebase_admin import storage
+
+from files.models import UploadedFile
+
+
+@receiver(post_delete, sender=UploadedFile)
+def delete_uploaded_file_blob(sender, instance: UploadedFile, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `sender`

(ARG001)